### ISSUE TITLE
`test_snapshot` should accept ExUnit's `context` as a parameter

### ISSUE THREAD

**Issue by jfrolich** [DCzajkowski/snapshy#3]
Great package. Actually found out about this before I wanted to write something similar. Works great and saved me a bunch of time!

One minor thing is that it would be nice for `test_snapshot` to also accept context as an argument.

**Comment by johngeorgewright** [comment]
Also just came across this package and immediately hit this problem.

```elixir
test_snapshot "with context", context do
  Api.do_something(context[:some_stuff])
end
```

```
** (CompileError)  undefined function test_snapshot/3
```

**Comment by DCzajkowski** [comment]
Fixed in #4. Thank you @johngeorgewright for contributing!

**Comment by DCzajkowski** [comment]
The new version 0.2.4 [has been released](https://hex.pm/packages/snapshy/0.2.4)! 🎉
